context_manager/instruction.md

```diff
@@ -0,0 +1,37 @@
+Context managers are a way of allocating and releasing some sort of resource exactly where you need it. The most widely used example of context managers is the 'with' statement.
```

Comment:
let's instructions.md to be only instructions.
I mean here we should have instructions  what student has to do.
No need information about  Context manager here
only steps:
1. do this 
2. do this